content/_data/changelogs/lts.yml

```diff
@@ -12986,6 +12986,495 @@
       Upgrade to Winstone 8.10 and Jetty 12.0.22.
       Jetty upgrade includes multiple bug fixes and most recent release of Eclipse Jetty 12.
 
+- version: "2.516.1"
+  date: 2025-07-23
+  lts_predecessor: "2.504.3"
+  lts_baseline: "2.516"
+  changes: # compared to lts_baseline 2.516 - extracted from the RC commit(s)
+
+  - type: rfe
+    category: rfe
+    pull: 10772
+    issue: 75818
+    authors:
+      - scherler
+    pr_title: "[JENKINS-75818] add event listener to trigger compute"
+    message: |-
+      Add window listener <code>computeOverflow</code> to trigger a new compute of the breadcrumbs.
+  - type: bug
+    category: bug
+    pull: 10804
+    issue: 75853
+    authors:
+      - timja
+    pr_title: "[JENKINS-75853] - Remove active look of hamburger button"
+    message: |-
+      Improve UI of secondary actions in the header.
+  - type: bug
+    category: bug
+    pull: 10786
+    issue: 75834
+    authors:
+      - scherler
+    pr_title: "[JENKINS-75834] make sure only one instance of the dropdown is visible"
+    message: |-
+      Prevent multiple dropdowns appearing in breadcrumbs.
+  - type: bug
+    category: bug
+    pull: 10791
+    issue: 75832
+    authors:
+      - jtnord
+    pr_title: "[JENKINS-75832] Support actions with Badges in the Hamburger Menu"
+    message: |-
+      The hamburger menu now correctly displays the Actions with Alerts.
+  - type: bug
+    category: bug
+    pull: 10800
+    issue: 75849
+    authors:
+      - alecharp
+    pr_title: "[JENKINS-75849] Makes scriptConsole layout configurable"
+    message: |-
+      Make script console layout configurable.
+
+  lts_changes: # compared to lts_predecessor 2.504.3 (selected by personal review)
+
+  - type: major rfe
+    category: major rfe
+    pull: 10351
+    issue: 75350
+    authors:
+      - alecharp
+    pr_title: "[JENKINS-75350] Displays plugin health score in Plugin Manager"
+    message: |-
+      Display plugin health score in the Plugin Manager.
+  - type: major rfe
+    category: major rfe
+    pull: 10245
+    authors:
+      - janfaracik
+    pr_title: Redesign the Jenkins header
+    message: |-
+      The Jenkins header is redesigned.
+  - type: major bug
+    category: bug
+    pull: 10647
+    issue: 75664
+    authors:
+      - jglick
+    pr_title: "[JENKINS-75664] Revert call to `BoundObjectTable#releaseMe`"
+    message: |-
+      A change to lazy rendering in 2.510 caused a regression in certain GUI configuration scenarios.
+      An example scenario would be the addition of two build steps of the same type, like <code>Execute shell</code>, to a freestyle project at the same time.
+      The workaround is to add one builder, save, reload the page, and then add the next.
```

Comment:
This bug was introduced in Jenkins 2.510 and was fixed in Jenkins 2.511 as far as I can tell from reading the [Jira issue](https://issues.jenkins.io/browse/JENKINS-75664).  Since 2.504 was the preceding LTS baseline, that means no LTS user ever saw this issue.  We don't need to document this issue in the 2.516.1 changelog.
```suggestion
```